osf/external/spam/tasks.py

```diff
@@ -0,0 +1,26 @@
+import re
```

Comment:
nit: not sure how this classifies as "external"

Reply:
The idea is that all the spam services, (oopspam, askismet etc) should be together and they all are external. I didn't expect a spam service to be an internal service.

Reply:
But none of the existing spam things are in this location. You created an `external/spam` directory *just for* an internal task. Can you put this in something like `osf/utils/spam.py` instead?